src/Components/ExpandableTags.js

```diff
@@ -0,0 +1,57 @@
+import { Button } from "@mui/material/";
+import Box from "@mui/material/Box";
+import Link from "@mui/material/Link";
+import useTheme from "@mui/material/styles/useTheme";
+import { useState } from "react";
+
+export default function ExpandableTags({ items, compactItems }) {
```

Comment:
Nit: It doesn't look like the `compactItems` parameter is used here, can we delete it?  Or will it be used elsewhere?

Reply:
Good point! I've removed it from the interface and made it a const in the file.